modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/EpisodeDao.kt

```diff
@@ -376,5 +376,5 @@ abstract class EpisodeDao {
         WHERE episodes.last_playback_interaction_date IS NOT NULL AND episodes.last_playback_interaction_date > :fromEpochMs AND episodes.last_playback_interaction_date < :toEpochMs
         """
     )
-    abstract fun findEpisodesCountInListeningHistory(fromEpochMs: Long, toEpochMs: Long): Flow<Int>
+    abstract suspend fun findEpisodesCountInListeningHistory(fromEpochMs: Long, toEpochMs: Long): Int
```

Comment:
I switched this to get the result once instead of a Flow as we only need to know the value at the start of the flow.